js/background.js

```diff
@@ -993,7 +993,9 @@
       let friendList = contacts;
       if (friendList !== undefined) {
         friendList = friendList.filter(
-          friend => friend.type === 'direct' && !friend.isMe
+          friend =>
+            (friend.type === 'direct' && !friend.isMe) ||
```

Comment:
is `friend` a conversation here? or is it a custom object?

Reply:
this is a ConversationType. why?

Reply:
you can just do `isPrivate` for `direct` and `isClosedGroup`